app/lib/seeds.rb

```diff
@@ -127,18 +135,18 @@ def create_contact(params = {})
     Contact.create!(params)
   end
 
-  def create_token(user, roles, token_params: {}, authorization_request_params: {})
+  def create_token(user, scopes, token_params: {}, authorization_request_params: {})
     authorization_request = create_authorization_request(authorization_request_params.merge(user:))
 
-    token = JwtAPIEntreprise.create!(
-      JwtAPIEntreprise.default_create_params
+    token = Token.create!(
+      Token.default_create_params
         .merge(token_params)
         .merge(
           authorization_request:
         )
     )
 
-    token.roles = roles
+    token.update(scopes:)
```

Comment:
```suggestion
    token.update!(scopes:)
```

Sur des seeds on veut un shebang pour raise une erreur (et éviter les fails silencieux)